main.py

```diff
@@ -503,14 +503,17 @@ def handle_new_switch(self, event):
         """Create a new Device on the Topology.
 
         Handle the event of a new created switch and update the topology with
-        this new device.
+        this new device. Also notify if the switch is enabled.
         """
         switch = event.content['switch']
         switch.activate()
         log.debug('Switch %s added to the Topology.', switch.id)
         self.notify_topology_update()
         self.update_instance_metadata(switch)
         self.restore_network_status(switch)
+        if switch.is_enabled():
+            self.notify_switch_enabled(switch.id)
+            self.save_status_on_storehouse()
```

Comment:
Why saving the status here? Nothing has been updated inside the `if` clause.

Reply:
When Kytos is running with `-E` (Enable all), the administrative changes (E.g. Switch enabled) is not saved in the storehouse. But, I agree with you, this may not be the best place to store that change.

Reply:
This modification was discarded in the last commit. I'll do another issue and pull request to check persistence when kytos was running with `-E`